ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -135,10 +135,13 @@ then
 		then
 		fi
 
+		# We don't start this mag on the Pixhawk 2, because we do not realistically
```

Comment:
This is also a 2nd accel. Isn't the right fix to change the start order such that the external units come first?